STANDARD.md

```diff
@@ -95,7 +95,7 @@ Each file's *root* group (path `/`) must at least contain the attributes:
       present standard (e.g. fields on an unstructured mesh),
       this can be done be storing this data within a path that *is not*
       of the form given by `basePath` (e.g. `/extra_data`). In this
-      way, the openPMD parsing tools will not parse this additional data. 
+      way, the openPMD parsing tools will not parse this additional data.
```

Comment:
although this EOL whitespace is annoying and needs to go, pls avoid changing lines that are not related to the change set so we can keep the chance for collisions with other PRs minimal